analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContext.cs

```diff
@@ -64,6 +64,9 @@ internal SonarAnalysisContext(AnalysisContext analysisContext, IEnumerable<Diagn
     public override bool TryGetValue<TValue>(SourceText text, SourceTextValueProvider<TValue> valueProvider, out TValue value) =>
         analysisContext.TryGetValue(text, valueProvider, out value);
 
+    internal static bool LegacyIsRegisteredActionEnabled(IEnumerable<DiagnosticDescriptor> diagnostics, SyntaxTree tree) =>
```

Comment:
Or we could rename it to `IsAllowedByOldSonarLint` and having a comment with what version we consider old or legacy will be awesome.

Reply:
I had a pending question to Duncan about exact version where it was removed (if I'm not mistaken that it was). I'll update/rename it once I have the answer (hopefully in this PR)

Reply:
For archeology: It was present in SL 5.5 and removed in SL 6.0